test/e2e/multicast_test.go

```diff
@@ -45,9 +45,11 @@ func skipIfMulticastDisabled(tb testing.TB, data *TestData) {
 	}
 }
 
+var transportInterface string
 var igmpQueryType = int32(0x11)
 
 func TestMulticast(t *testing.T) {
+	skipIfAntreaIPAMTest(t)
```

Comment:
Doesn't we add such support, why not running it when AntreaIPAM is enabled?

Reply:
updated